src/bug-filing/common/get-issue-details-markdown.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { IssueUrlCreationUtils } from './issue-filing-url-string-utils';
+
+export type IssueDetailsGetter = (
+    stringUtils: IssueUrlCreationUtils,
+    environmentInfo: EnvironmentInfo,
+    data: CreateIssueDetailsTextData,
+) => string;
+
+export const getIssueDetailsMarkdown = (
```

Comment:
Declare type explicitly to detect any future signature divergence?
```suggestion
export const getIssueDetailsMarkdown: IssueDetailsGetter = (
```